src/app/add-problem-statement/add-problem-statement.page.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-add-problem-statement',
+  templateUrl: './add-problem-statement.page.html',
+  styleUrls: ['./add-problem-statement.page.scss'],
+})
+export class AddProblemStatementPage implements OnInit {
+  headerConfig:any = {
+    showBackButton:true
+  }
+  showInput = false;
+  problemStatement: string = '';
+  selectedOption: string = '';
+  options = ['Option 1', 'Option 2', 'Option 3', 'Option 4'];
```

Comment:
hope this data will be getting from api in the next pr

Reply:
yes